extensions/wikia/AdEngine/js/recovery/message.js

```diff
@@ -12,21 +13,37 @@ define('ext.wikia.adEngine.recovery.message', [
 	'use strict';
 
 	var logGroup = 'ext.wikia.adEngine.recovery.message',
+		wikiaTopAdsId = 'WikiaTopAds',
 		wikiaRailId = 'WikiaRail';
 
-	function addRecoveryCallback() {
-		recoveryHelper.addOnBlockingCallback(function () {
-			recover();
-		});
+	function createMessage(uniqueClassName) {
+		var className = 'recovered-message',
+			div = doc.createElement('div');
+
+		div.textContent = 'Hello world!';
+		div.classList.add(className, className + '-' + uniqueClassName);
```

Comment:
Since we [support IE11](http://community.wikia.com/wiki/Help:Supported_Browsers) we [can't use the 2nd parameter](http://caniuse.com/#feat=classlist) and we need to implement toggling again :(